src/clr_bundle_add.c

```diff
@@ -53,13 +56,19 @@ static void print_help(void)
 }
 
 static const struct option prog_opts[] = {
-	{ "skip-optional", no_argument, &skip_optional_bundles, 1 },
-	{ "skip-diskspace-check", no_argument, &skip_diskspace_check, 1 },
+	{ "skip-optional", no_argument, 0, FLAG_SKIP_OPTIONAL },
+	{ "skip-diskspace-check", no_argument, 0, FLAG_SKIP_DISKSPACE_CHECK },
 };
 
 static bool parse_opt(int opt, UNUSED_PARAM char *optarg)
 {
 	switch (opt) {
+	case FLAG_SKIP_OPTIONAL:
+		skip_optional_bundles = true;
+		return true;
+	case FLAG_SKIP_DISKSPACE_CHECK:
+		skip_diskspace_check = optarg ? strtobool(optarg) : true;
```

Comment:
you forgot to also change this one to:
skip_diskspace_check = true;

Reply:
oops. fixed